src/test/java/core/basesyntax/dao/TransactionParserTest.java

```diff
@@ -0,0 +1,108 @@
+        FruitTransaction firstFruitTransaction = new FruitTransaction();
+        firstFruitTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        firstFruitTransaction.setFruit("banana");
+        firstFruitTransaction.setQuantity(20);
+        FruitTransaction secondFruitTransaction = new FruitTransaction();
+        secondFruitTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        secondFruitTransaction.setFruit("apple");
+        secondFruitTransaction.setQuantity(100);
+        FruitTransaction thirdFruitTransaction = new FruitTransaction();
+        thirdFruitTransaction.setOperation(FruitTransaction.Operation.SUPPLY);
+        thirdFruitTransaction.setFruit("banana");
+        thirdFruitTransaction.setQuantity(100);
+        List<FruitTransaction> defaultList = new ArrayList<>();
+        defaultList.add(firstFruitTransaction);
+        defaultList.add(secondFruitTransaction);
+        defaultList.add(thirdFruitTransaction);
+        return defaultList;
+    }
+
+    public List<FruitTransaction> fruitTransactionForSecondCase() {
+        FruitTransaction firstFruitTransaction = new FruitTransaction();
+        firstFruitTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        firstFruitTransaction.setFruit("apple");
+        firstFruitTransaction.setQuantity(40);
+        FruitTransaction secondFruitTransaction = new FruitTransaction();
+        secondFruitTransaction.setOperation(FruitTransaction.Operation.BALANCE);
+        secondFruitTransaction.setFruit("orange");
+        secondFruitTransaction.setQuantity(80);
+        FruitTransaction thirdFruitTransaction = new FruitTransaction();
+        thirdFruitTransaction.setOperation(FruitTransaction.Operation.SUPPLY);
+        thirdFruitTransaction.setFruit("orange");
+        thirdFruitTransaction.setQuantity(10);
+        FruitTransaction fourthFruitTransaction = new FruitTransaction();
+        fourthFruitTransaction.setOperation(FruitTransaction.Operation.SUPPLY);
+        fourthFruitTransaction.setFruit("apple");
+        fourthFruitTransaction.setQuantity(10);
+        List<FruitTransaction> defaultList = new ArrayList<>();
+        defaultList.add(firstFruitTransaction);
+        defaultList.add(secondFruitTransaction);
+        defaultList.add(thirdFruitTransaction);
+        defaultList.add(fourthFruitTransaction);
+        return defaultList;
+    }
+
+    @Test
+    public void parseTransaction_firstCase_Ok() {
+        TransactionParser transactionParser = new TransactionParserImpl();
+        String data = "b,banana,20\n"
+                + "b,apple,100\n"
+                + "s,banana,100\n";
+        List<FruitTransaction> expectedListTransaction = fruitTransactionForFirstCase();
+        List<FruitTransaction> actualTransactionList = transactionParser.parseTransactions(data);
+        assertEquals(expectedListTransaction, actualTransactionList);
+    }
+
+    @Test
+    public void parseTransaction_secondCase_Ok() {
```

Comment:
Why do you need two OK cases? If you really need two separate - describe them somehow, what are you testing? (not simple **first** and **second**)